pyproject.toml

```
@@ -120,3 +121,14 @@ testfixtures = "*"
 tuspy = "*"
 twill = "*"
 watchdog = "*"
+
+[tool.black]
+line-length = 120
+target-version = ['py37']
+include = '\.pyi?$'
+extend-exclude = '''
+/(
```

```suggestion
^/(
```

Otherwise directories like `lib/galaxy/tools` are excluded.